### PR TITLE
Refactor Code

### PR DESCRIPTION
**Pull request by abbasali1994**
Ticket(s):
-  Closes : [Refactor propMeta ternary custom.ts helper#1837](https://github.com/AngelProtocolFinance/angelprotocol-web-app/issues/1837)
- Closes: [Refactor getCWs custom.ts helper#1836](https://github.com/AngelProtocolFinance/angelprotocol-web-app/issues/1836)

## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

### CODE REVIEW

File: src/services/juno/custom/helpers/admin-resource.ts [hunk 1]
```diff
@@ -10,14 +10,21 @@ import { customApi } from "../custom";
...
+
 export function getCWs(id: number) {
-  //TODO: atm, only two admin types, refactor this once > 2
-  //charities doesn't have hardcoded cws, so only test for AP_ID && REVIEWER_ID
-  const cw3 = id === AP_ID ? contracts.cw3ApTeam : contracts.cw3ReviewTeam;
-  const cw4 =
-    id === AP_ID ? contracts.cw4GrpApTeam : contracts.cw4GrpReviewTeam;
-  const type: AdminResources["type"] = id === AP_ID ? "ap" : "review";
-  return { cw3, cw4, type };
+  return isAp(id) ? CONTRACTS.AP : CONTRACTS.REVIEWER;
```
Comment (ap-justin) [inline src/services/juno/custom/helpers/admin-resource.ts:27]: we could also get rid of functions `getCWs` and `isAp` entirely

and make use of just the `constant`
```ts
type Admins = AdminResources["type"];
const _charity: Admins = "charity";
type CWs = {
  [key: number]:
    | {
        cw3: string;
        cw4: string;
        type: Exclude<Admins, typeof _charity>;
      }
    | undefined;
};

export const apCWs: CWs = {
  [AP_ID]: {
    cw3: contracts.cw3ApTeam,
    cw4: contracts.cw4GrpApTeam,
    type: "ap",
  },
  [REVIEWER_ID]: {
    cw3: contracts.cw3ApTeam,
    cw4: contracts.cw4GrpApTeam,
    type: "ap",
  },
};
```

and use as
```ts
 const numId = idParamToNum(args.endowmentId);
        const AP = apCWs[numId];
        /** special case for ap admin usage */
        if (AP) {
          const { cw3, cw4, type } = AP;
          //...make use of values
         }
```

**Review by ap-justin — CHANGES_REQUESTED** [review]

File: src/services/juno/custom/helpers/admin-resource.ts (hunk 1 above)
Comment (abbasali1994) [reply src/services/juno/custom/helpers/admin-resource.ts]: Implemented aabove change

**Review by ap-justin — DISMISSED** [review]

File: src/services/juno/custom/helpers/admin-resource.ts [hunk 2]
```diff
@@ -10,19 +10,30 @@ import { customApi } from "../custom";
...
+export const apCWs: CWs = {
+  [AP_ID]: {
+    cw3: contracts.cw3ApTeam,
+    cw4: contracts.cw4GrpApTeam,
+    type: "ap",
+  },
+  [REVIEWER_ID]: {
+    cw3: contracts.cw3ApTeam,
+    cw4: contracts.cw4GrpApTeam,
+    type: "ap",
```
Comment (misicnenad) [inline src/services/juno/custom/helpers/admin-resource.ts:34]:
```suggestion
    type: "review",
```

File: src/services/juno/custom/helpers/admin-resource.ts [hunk 3]
```diff
@@ -10,19 +10,30 @@ import { customApi } from "../custom";
...
-  return id === AP_ID || id === REVIEWER_ID;
-}
+export const apCWs: CWs = {
+  [AP_ID]: {
+    cw3: contracts.cw3ApTeam,
+    cw4: contracts.cw4GrpApTeam,
+    type: "ap",
+  },
+  [REVIEWER_ID]: {
+    cw3: contracts.cw3ApTeam,
```
Comment (misicnenad) [inline src/services/juno/custom/helpers/admin-resource.ts:32]:
```suggestion
    cw3: contracts.cw3ReviewTeam,
```

File: src/services/juno/custom/helpers/admin-resource.ts [hunk 4]
```diff
@@ -10,19 +10,30 @@ import { customApi } from "../custom";
...
+    cw4: contracts.cw4GrpApTeam,
```
Comment (misicnenad) [inline src/services/juno/custom/helpers/admin-resource.ts:33]:
```suggestion
    cw4: contracts.cw4GrpReviewTeam,
```

**Review by misicnenad — CHANGES_REQUESTED** [review]

File: src/services/juno/custom/helpers/admin-resource.ts (hunk 2 above)
Comment (misicnenad) [reply src/services/juno/custom/helpers/admin-resource.ts:34]: Using AP params for `REVIEWER_ID` is a mistake in the example given in https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1856#discussion_r1115678467, should use Reviewer params.

File: src/services/juno/custom/helpers/admin-resource.ts (hunk 3 above)
Comment (misicnenad) [reply src/services/juno/custom/helpers/admin-resource.ts:32]: Using AP params for `REVIEWER_ID` is a mistake in the example given in https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1856#discussion_r1115678467, should use Reviewer params.

File: src/services/juno/custom/helpers/admin-resource.ts (hunk 4 above)
Comment (misicnenad) [reply src/services/juno/custom/helpers/admin-resource.ts:33]: Using `contracts.cw4GrpApTeam` for `REVIEWER_ID` is a mistake in the example given in https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/1856#discussion_r1115678467, should use `contracts.cw4GrpReviewTeam`.

File: src/services/juno/custom/helpers/admin-resource.ts (hunk 3 above)
Comment (abbasali1994) [reply src/services/juno/custom/helpers/admin-resource.ts]: done

File: src/services/juno/custom/helpers/admin-resource.ts (hunk 4 above)
Comment (abbasali1994) [reply src/services/juno/custom/helpers/admin-resource.ts]: done

**Pull request merged**